### PR TITLE
2.4.4 patch release

### PR DESCRIPTION
**Pull request by jfy133**
Single change: fixing bed files being consumed.

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](<https://github.com/>nf-core/eager/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/eager _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,18 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [dev]
```
Comment (jfy133) [inline CHANGELOG.md:6]:
```suggestion
## [2.4.4] - 2020-04-07
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,18 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.4.4] - 2020-04-07
```
Comment (jfy133) [inline CHANGELOG.md:6]:
```suggestion
## [2.4.4] - 2022-04-07
```

**Review by TCLamnidis — APPROVED** [review]
Not sure why adding .collect() to single file channels fixes the issue, but i trust that it works. 
LGTM

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,18 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.4.4] - 2022-04-07
```
Comment (jfy133) [inline CHANGELOG.md:6]:
```suggestion
## [2.4.4] - 2022-04-08
```

**Comment by jfy133** [comment]
> Not sure why adding .collect() to single file channels fixes the issue, but i trust that it works. LGTM

https://www.nextflow.io/docs/latest/channel.html#value-channel

**Pull request merged**